### PR TITLE
Auto respond with "%name% has chat disabled" if chat disabled.

### PR DESCRIPTION
**Pull request by rapito**
### Auto respond with "%name% has chat disabled" if chat disabled.
This only happens on singles, on Teams it wont come up. 
image

##### Bug Fixes: 
- Prevent messages on same frame to hide each other
- Japenese names now work as expected
- Players in Teams can now chat without issues

### CODE REVIEW

**Comment by JLaferri** [comment]
Didn't look very hard but I didn't see it in my skim: does this only happen in singles? Probably shouldn't get sent in teams lobbies.

**Comment by rapito** [comment]
> Didn't look very hard but I didn't see it in my skim: does this only happen in singles? Probably shouldn't get sent in teams lobbies.

_Fixed._

**Comment by rapito** [comment]
@JLaferri This PR now also includes the fix for users on Teams not being able to send chat messages after a couple of them are sent. 
(https://github.com/project-slippi/Ishiiruka/pull/256/commits/213de1adc6227637f8f528749260d6598a323d33)

I also removed the logs you mentioned @NikhilNarayana

**Comment by rapito** [comment]
This PR now includes a patch that fixes Japanese Names when using Quick Chat.

 https://twitter.com/Kounotori_ssbm/status/1370050876744769538
 
image

File: Source/Core/Core/Slippi/SlippiPremadeText.h [hunk 1]
```diff
@@ -1,5 +1,6 @@
-#pragma once
+﻿#pragma once
 
+#include "../../Common/StringUtil.h"
```
Comment (JLaferri) [inline Source/Core/Core/Slippi/SlippiPremadeText.h]: is a relative path needed? dont remember seeing this anywhere else this is included uses `#include "Common/StringUtil.h"`

**Pull request merged**